Dockerfile

```
@@ -0,0 +1,10 @@
+FROM python:3-alpine
+
+WORKDIR /blc
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY blc ./blc
```

```suggestion
COPY requirements.txt blc ./
RUN pip install --no-cache-dir -r requirements.txt
```

Is his not working ? 🤔

---

no

---

ok 👍🏽